pairAndIntroduce.ts

```diff
@@ -9,5 +24,38 @@ const app = new App({
   appToken: process.env.SLACK_APP_TOKEN // add this
 });
 
-const pairs = generatePairs(app);
+export const startConversations = async (app: App) => {
+  const pairs = await generatePairs(app);
+  // iterate through all pairs to open a DM and send an intro message
+  for (const pair of pairs) {
+    let thirdUser = "";
+    // account for potential group of 3
+    if (pair.length > 2) {
+      thirdUser = "," + pair[2];
+    }
+    // generate users string
+    const users: string = `${pair[0]},${pair[1]}${thirdUser}`;
```

Comment:
i think we can generalize this to work for any length of `pairs` vs a specific case for group of 3 (even though realistically a pair would only have 2 or 3).
maybe we can create `users` with [the join method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join):
```suggestion
    // generate users string
    const users: string = pair.join();
```

Reply:
omg join so powerful